### PR TITLE
Pop window feature

### PR DESCRIPTION
**Pull request by zied2112**
In this Milestone we coded the following:
- Implement the work section using the js file
- Adding and Styling the pop Window

### CODE REVIEW

File: js/projectData.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
+const data = [
+  {
+    title: 'Keeping track of hundreds of components',
+    src: '../images/Snapshoot Portfolio.jpg',
+    description:
```
Comment (AmaduKamara) [inline js/projectData.js]: - [ ] When the data is fetched and displayed, the images appeared broken and it's a result of the paths specified in your array of objects. I removed one dot(.) from the double dots(../) path you used and the images came back to life. Kindly fix that 👍 

image

File: js/index.js [hunk 2]
```diff
@@ -13,4 +18,69 @@ closeBtn.addEventListener('click', () => {
...
+  projectBtn.addEventListener('click', () => {
+    document.getElementById('popup').scrollIntoView();
+    projectContainer.style.display = 'none';
+    popup.style.display = 'flex';
+    const y = popup.getBoundingClientRect().top + window.scrollY;
+    window.scroll({
+      top: y,
+      behavior: 'smooth',
+    });
+    popup.innerHTML += `<div class="popup-project-container">
```
Comment (AmaduKamara) [inline js/index.js:51]: - [ ] Kindly adjust the paths for your images fetched from the array of objects
- [ ] I see you forgot to place the close ( X ) on the pop-up windows. Please fix that as it is there to allow the user to close the modal window by clicking that X icon. 👍

**See the design**
image

**Review by AmaduKamara — CHANGES_REQUESTED** [review]
# Status: Required Changes ♻️♻️♻️ 

Hi @zied2112,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✅ You used GitHub workflow correctly
✅ No linter error
✅ Dynamically fetched popup modal contents 👍

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Comment by zied2112** [comment]
Fixed 💯 
image
The Icon is in White on the mobile version as you see 
image

File: js/index.js [hunk 3]
```diff
@@ -1,6 +1,11 @@
+import data from './projectData.js';
+
 const menuBtn = document.getElementById('menu-btn');
 const closeBtn = document.getElementById('close-btn');
 const menu = document.getElementById('menu');
+const projectContainer = document.getElementById('projects-container');
+
+const popup = document.getElementById('popup');
 menuBtn.addEventListener('click', () => {
   // console.log('Clicked');
```
Comment (Munsa1) [inline js/index.js]: - [ ] Kindly remove commented code as it makes your codebase unclean.

File: js/index.js [hunk 4]
```diff
@@ -1,6 +1,11 @@
+import data from './projectData.js';
+
 const menuBtn = document.getElementById('menu-btn');
 const closeBtn = document.getElementById('close-btn');
 const menu = document.getElementById('menu');
+const projectContainer = document.getElementById('projects-container');
+
+const popup = document.getElementById('popup');
```
Comment (Munsa1) [inline js/index.js:8]: - [ ] Kindly make the close button visible to the mobile version of the pop-up card.

Your design
image

Figma design
image

- [ ] Kindly align the buttons in the mobile version to match the given design
- [ ] Kindly style the languages to match the given design

Your design
image

Figma design
image

**Review by Munsa1 — CHANGES_REQUESTED** [review]
Hi @zied2112, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
:heavy_check_mark: Descriptive Readme
:heavy_check_mark: Passing Linters
:heavy_check_mark: Correct Gitflow

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: js/index.js (hunk 4 above)
Comment (zied2112) [reply js/index.js:8]: Sorry, but If I make it visible by changing the style it will not correspond to the template design!

Comment (zied2112) [reply js/index.js:8]: image
It will be okay like this ?

**Review by aimemalaika — APPROVED** [review]
Hi @zied2112 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 🎉

- No linter errors. ✔️
- GitHub flow is followed. ✔️
- Professional README file. ✔️
- PR has a good title and summary. ✔️

133591850-fdcb7dd3-dd7a-4274-81e7-0e59434148f7

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**